src/routes/calculator/modules/calculator.js

```diff
@@ -0,0 +1,29 @@
+//actions
+
+export const calcState = {
+    calValue : 0,
+    calBtns : ["1", "2", "3", "C ","4", "5", "6", "=", "7", "8", "9", ".", "+", "0", "-", "*", "/"]
+}
+
+// reducer
+export const calcReducer = (state = calcState, action ) =>{
+    switch(action.type){
+        case 'ADD_ELEM':
```

Comment:
@priyanka0901 This action type is confusing, `ADD_ELEM` is actually concatinating the string, but it sounds like it is for `addition`.